tests/RobotFramework/tests/tedge/tedge_config_get.robot

```diff
@@ -66,10 +66,20 @@ Custom Setup
 
 Check known tedge environment settings
     [Arguments]    ${ENV_NAME}    ${KEY_NAME}    ${VALUE}
-    ${output}=    Execute Command    cmd=env ${ENV_NAME}=${VALUE} tedge config get ${KEY_NAME}
-    Should Be Equal    ${output}    ${VALUE}\n
+    ${stdout}    ${stderr}=    Execute Command
+    ...    cmd=env ${ENV_NAME}=${VALUE} tedge config get ${KEY_NAME}
+    ...    stdout=${True}
+    ...    stderr=${True}
+    ...    retries=1
+    Should Be Equal    ${stdout}    ${VALUE}\n
```

Comment:
nit: I think `Execute Command    strip=True` could be used here to remove that extra `\n`

Reply:
Though we're checking the exact output here, so we want to be sure that we're only outputting multiple newline chars (which would be masked if using the strip option)